packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md

```diff
@@ -0,0 +1,37 @@
+-   Button **labels** should be **short** and use **“trigger words”**.
+-   As much as possible, the **first word of the label** should be an **action verb** (e.g., the label for a button that allows the user to add a filter could be “Add Filter").
+
+### Alterative Texts
+
+-   When a **button includes only an icon** (no label), the **label** should be added in the **alternative text element in the HTML** and appear in a **tooltip** when the cursor is above the button.
+-   An **icon with a label does not need alternative text**. In this case, the icon is a visual help to identify the purpose faster and filling it will cause a screen reader to repeat the text twice.
+
+## Variations
+
+The Button Type choice is defined by the function of the button (e.g. primary or a secondary action), the number of actions available in the screen (e.g. only one, two or more actions) and its location (e.g. displayed in a header, action bar, page or footer).
+
+| Type      | Purpose                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Primary   | Primary Buttons are used to enable the most important actions, that is those that lead to the completion of the main task (e.g., Save, Continue, Submit), or strong call to actions. Ideally, there should be only one Primary Button per screen to avoid dilluting the user's attention.                                                                                                                                                                                                  |
+| Secondary | Secondary Buttons are used for main actions that users might want to execute but that are not in direct relation to the main task that is to be done in the screen or interface.                                                                                                                                                                                                                                                                                                           |
```

Comment:
Suggestion:
Secondary Buttons are used for main actions -> **Secondary Buttons** enable main actions